### DISCUSSION TITLE
页码

### DISCUSSION THREAD

**Discussion by VickyCas** [xjtu-blacksmith/easymcm#9] [Q&A] [answered]
您好 您的代码很有用 今年的美赛要求是目录页也需要有页数标注 请问如何标呢 我用的就是2021的您的latex代码 谢谢！

**Comment by Simon9Zhou** [comment]
目录加上页眉的页码显示：
将easymcm.sty文件中，最后一部分代码
% 目录生成
中的
\thispagestyle{empty}
改成
\thispagestyle{fancy}

但是同样有另一个问题，今年美赛要求整体不超过25页，包括摘要、目录、参考文献和附录，此处页码从目录开始算1，摘要没有在总页数里面，还请作者修改

**Reply by Simon9Zhou** [reply]
我又试了一下如何让摘要页计算进页码总数，
在easymcm.sty文件中
% Summary Sheet 生成
部分
\setcounter{page}{1}}
的1改成2，再编译两次
应该可以让页码都加一

**Comment by xjtu-blacksmith** [comment]
我刚刚更新了 v5.2 版本，已经解决了页码计数的问题。

在官方说明中未见目录需要标注页眉的要求，请您帮忙找到说明中的原文，确认之后我会及时将这项内容加入模板之中。

**Comment by xjtu-blacksmith** [comment, marked as answer]
补记一下：去年（2021 年）发布的 v5.2.1 版本中，已经解决了这个贴子提到的另一个问题——目录页眉的问题。之后版本中亦不存在类似问题。

**Comment by Simon9Zhou** [comment]
谢谢你的来信，我已经收到并将尽快回复！Thank you for your email, I will reply as soon as possible.
